deploy/template.yaml

```diff
@@ -0,0 +1,290 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  ipv-core-back
+
+Globals:
+  Function:
+    Timeout: 20
+
+Parameters:
+  Environment:
+    Type: String
+
+Resources:
+
+  IPVCoreAPI:
+    Type: AWS::Serverless::Api
+    Properties:
+      Name: IPV Core API Gateway
+      StageName: !Sub ${Environment}
+
+  IPVAccessTokenFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "ipv-access-token-${Environment}"
+      Handler: uk.gov.di.ipv.core.accesstoken.AccessTokenHandler::handleRequest
+      Runtime: java11
+      PackageType: Zip
+      CodeUri: ../lambdas/accesstoken
+      Architectures:
+        - x86_64
+      MemorySize: 512
+      Environment:
+        Variables:
+          AUTH_CODES_TABLE_NAME: !Select [1, !Split ['/', !GetAtt AuthCodesTable.Arn]]
+          ACCESS_TOKENS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt AccessTokensTable.Arn]]
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref AccessTokensTable
```

Comment:
It looks like this returns the table name? can it be used in place of `!Select [1, !Split ['/', !GetAtt AuthCodesTable.Arn]]`?

Reply:
From what I understand this is doing some magic in the background as you specify `TableName` so It wouldn't work where try specifying the table name for environment variables